ebpfsvc/api.cpp

```diff
@@ -30,5 +38,34 @@ ebpf_verify_program(
     UNREFERENCED_PARAMETER(logs_size);
     UNREFERENCED_PARAMETER(logs);
 
-    return EBPF_FAILED;
+    const char* report;
+    const char* error_message;
+    ebpf_result_t result = EBPF_SUCCESS;
+    int retVal = 0;
+    const char* path = "";
+    const char* section_name = "";
+
+    // Validate input
+    if (info == nullptr || info->byte_code_size == 0) {
```

Comment:
If these aren't permitted to be NULL, why not make them [[ref]](https://docs.microsoft.com/en-us/windows/win32/midl/ref) in the IDL? Then RPC will reject it if the pointer is NULL